Library/Homebrew/extend/fileutils.rb

```diff
@@ -67,6 +105,7 @@ def mkdir(name, &_block)
     # @private
     class Entry_
       alias_method :old_copy_metadata, :copy_metadata
+
```

Comment:
I'd prefer if the PR didn't introduce empty lines unrelated to the changed code.